python/tk_desktop2/websockets/websockets_connection.py

```diff
@@ -39,6 +40,10 @@ class WebsocketsConnection(object):
     _legacy_site_warning_displayed = False
     _legacy_user_warning_displayed = False
 
+    # Pre-compiled shotgunlocalhost.com/localhost regex matcher
+    # You can play with it here: https://regex101.com/r/7n3JIp/3
+    localhost_re = re.compile(r"^https?://(?:shotgunlocalhost\.com|localhost):[0-9]{1,5}$")
```

Comment:
What about http? Please add a comment explaining why we are excluding non-encrypted local comms!

Reply:
If i understand this correctly, I think we should we also allow `127.0.0.1` as a valid local address. Again, if not, i think it is helpful to add a comment to explain why we have excluded it!

Reply:
https? mean http or https (look at the URL)
I can add 127.0.0.1 to the regex.